backend/filldb.py

```diff
@@ -0,0 +1,190 @@
+import argparse
+import random
+import string
+import uuid
+from random import choice
+from typing import List
+
+from oasst_backend.api.deps import api_auth
+from oasst_backend.database import engine
+from oasst_backend.models import ApiClient
+from oasst_backend.prompt_repository import PromptRepository
+from oasst_shared.schemas.protocol import User as ProtocolUser
+from sqlmodel import Session
+
+
+# TODO: Should put settings into False
+class FillDb:
+    """Class object which fills the database: Api Client, Users & Messages."""
```

Comment:
It says "... & Messages" but I couldn't find the code that actually inserts messages? Is this still open?